imap-listener/src/healthcheck.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Health Check Server
+ */
+
+import http, { type IncomingMessage, type ServerResponse } from 'http';
+
+const port = parseInt(process.env.HEALTH_CHECK_PORT || '3100');
```

Comment:
Siehe oben. Wann würden wir den Port mal wechseln? => Sollte IMO in den Code unten.